integrations/segmentio/lib/index.js

```diff
@@ -315,10 +317,31 @@ Segment.prototype.normalize = function(message) {
   }
   if (this.options.addBundledMetadata) {
     var bundled = keys(this.analytics.Integrations);
+    var maybeBundledConfigIds = this.options.maybeBundledConfigIds
+
+    // Generate a list of bundled config IDs using the intersection of
+    // bundled destination names and maybe bundled config IDs.
+    var bundledConfigIds = []
```

Comment:
I think `var bundledConfigIds = []` should be defined outside of this conditional for clarity. This works currently because of use of `var`.

Reply:
Ahh. Good catch. 👍🏽

Reply:
I don't think so -- `msg._metadata.bundledConfigIds` (L343) is defined within the conditional. I'm also mimicking the scope of `bundled`.